webapp/src/features/intersections/assessments/lane-direction-of-travel-assessment.tsx

```diff
@@ -112,31 +123,21 @@ export const LaneDirectionOfTravelAssessmentCard = (props: {
   }
 
   return (
-    <Grid2 height="500px">
-      <Card sx={{ height: '100%', overflow: 'visible' }}>
+    <Grid2 sx={{ height: '100%', minHeight: assessment === undefined ? 200 : 500 }}>
```

Comment:
Question: What's the reasoning for setting minHeight and height here? Are both needed?

Reply:
If there are 2 cards next to each other, I want them to both fill the available space. But, I want them to be able to shrink if both are short, like this:
![dsahboard_tiny](https://github.com/user-attachments/assets/dde94c3e-3343-4c20-864e-de3ad3374fbc)